akka-docs/src/test/scala/docs/stream/operators/sink/Collection.scala

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2020 Lightbend Inc. <https://www.lightbend.com>
+ */
+
+package docs.stream.operators.sink
+
+import akka.actor.ActorSystem
+import akka.stream.scaladsl.{ Sink, Source }
+
+import scala.concurrent.{ ExecutionContextExecutor, Future }
+
+object Collection {
+  implicit val system: ActorSystem = ???
+  implicit val ec: ExecutionContextExecutor = system.dispatcher
+  def collectionExample: Future[Unit] = {
+    //#collection
+    val source = Source(1 to 100)
+    val result: Future[List[Int]] = source.runWith(Sink.collection[Int, List[Int]])
+    result.map(println)
```

Comment:
`Future.foreach` to only side effect. Better to take a lower number of elements, say 5, and add a comment after this showing exactly what will be printed.